spec/AsComponent.coffee

```diff
@@ -67,6 +67,19 @@ describe 'asComponent interface', ->
           ins.post new noflo.IP 'data', 'Bar'
           ins.post new noflo.IP 'data', 'Baz'
           ins.post new noflo.IP 'closeBracket', 'a'
+    describe 'with returned NULL', ->
+      func = (hello) ->
+        return null
+      it 'should be possible to componentize', (done) ->
+        component = -> noflo.asComponent func
+        loader.registerComponent 'ascomponent', 'sync-one', component, done
```

Comment:
Is this component name duplicated from other tests? Should be something like `'return-null'` ?

Reply:
Good point, fixing